docs/sphinx/using/examples/multi_gpu_workflows.rst

```diff
@@ -0,0 +1,270 @@
+
+-  **`nvidia`**: Single GPU based backend which accelerates quantum circuit
+   simulation on NVIDIA GPUs powered by cuQuantum.
+
+-  **`nvidia-mqpu`**: Enables users to program workflows utilizing
+   multiple quantum processors in parallel, enabled today by GPU emulation.
+
+-  **`nvidia-mgpu`**: Allows for scaling circuit simulation on multiple GPUs, which is beyond what is
+   feasible with any QPU today.
+
+-  **`density-matrix-cpu`**: Noisy simulations via density matrix
+   calculations. CPU only for now with GPU support coming soon.
+
+Below we explore how to effectively utilize multiple CUDA-Q targets with the same GHZ state preparation code
+
+.. code:: python
+
+    def ghz_state(qubit_count, target):
+        """A function that will generate a variable sized GHZ state (`qubit_count`)."""
+        cudaq.set_target(target)
+    
+        kernel = cudaq.make_kernel()
+    
+        qubits = kernel.qalloc(qubit_count)
+    
+        kernel.h(qubits[0])
+    
+        for i in range(1, qubit_count):
+            kernel.cx(qubits[0], qubits[i])
+    
+        kernel.mz(qubits)
+    
+        result = cudaq.sample(kernel, shots_count=1000)
+    
+        return result
+
+QPP-based CPU Backend
+~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    cpu_result = ghz_state(qubit_count=2, target="qpp-cpu")
+    
+    cpu_result.dump()
+
+
+.. parsed-literal::
+
+    { 00:475 11:525 }
+
+
+Acceleration via NVIDIA GPUs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Users will notice a speedup of up to **2500x** in executing the circuit below on
+NVIDIA GPUs vs CPUs.
+
+.. code:: python
+
+    gpu_result = ghz_state(qubit_count=25, target="nvidia")
+    
+    gpu_result.dump()
+
+
+.. parsed-literal::
+
+    { 0000000000000000000000000:510 1111111111111111111111111:490 }
+
+
+Multiple NVIDIA GPUs
+~~~~~~~~~~~~~~~~~~~~
+
+A :math:`n` qubit quantum state has :math:`2^n` complex amplitudes, each
+of which require 8 bytes of memory to store. Hence the total memory
+required to store a :math:`n` qubit quantum state is :math:`8` bytes
+:math:`\times 2^n`. For :math:`n = 30` qubits, this is roughly :math:`8`
+GB but for :math:`n = 40`, this exponentially increases to 8700 GB.
+
+If one incrementally increases the qubit count in their circuit, we
+reach a limit where the memory required is beyond the capabilities of a
+single GPU. The ``nvidia-mgpu`` target allows for memory from additional
+GPUs to be pooled enabling qubit counts to be scaled.
+
+Execution on the ``nvidia-mgpu`` backed is enabled via ``mpirun``. Users
+need to create a ``.py`` file with their code and run the command below
+in terminal:
+
+``mpirun -np 4 python3 test.py``
+
+where 4 is the number of GPUs one has access to and ``test`` is the file
+name chosen.
+
+Multiple QPUs
+~~~~~~~~~~~~~~
+
+The ``nvidia-mqpu`` backend allows for future multi-QPUs workflows made possible
+via GPU simulation today.
+
+Asynchronous Data Collection via Batching Hamiltonian Terms
```

Comment:
The [original page](https://nvidia.github.io/cuda-quantum/latest/examples/python/tutorials/multi_gpu_workflows.html#Asynchronous-data-collection-via-batching-Hamiltonian-terms) had 2 images showing a Hamiltonian breakdown to multiple terms, running on multiple devices.
Can you please also add it this page?
Images can be added to docs/sphinx/_static, see other pages on how they are referenced.